### ISSUE TITLE
"IMPORTANT - The tutorial will tell you to create a new folder where the Node project will lie"

### ISSUE THREAD

**Issue by montaser85** [adeshpande3/Facebook-Messenger-Bot#15]
Hello.
I am a bit confused about this part at the start of step number 7. Do I have to keep all the files created upto step 6 and the files created after  step 7 following your shared tutorial in the same folder and run the following commands:
git init
git add .
git commit --message "hello world"
heroku create
git push heroku master

Does the folder which will be committed to git should look like this?:
![11deds](https://user-images.githubusercontent.com/20108007/36114136-4b710446-1059-11e8-8a8d-602559b3ec4f.jpg)

**Comment by adeshpande3** [comment]
Yeah you can keep them in the same folder or you can have them separate from each other. The main thing you need to keep in mind (and this is mentioned in the [tutorial](https://github.com/jw84/messenger-bot-tutorial)) is that you have to create a new Node project and install all the dependencies, Node setup, etc. The important thing is that you have the Procfile and the index.js files (as you have above). Whether or not the files are in the same folder shouldn't matter, but do let me know if you have any issues. I may be forgetting something since it's been a while since I worked on this project.
